lib/bindings/python/src/dynamo/runtime/logging.py

```diff
@@ -94,7 +94,8 @@ def configure_dynamo_logging(
 
     # configure inference engine loggers
     configure_vllm_logging(dyn_level)
-    configure_sglang_logging(dyn_level)
+    if not os.environ.get("DYN_SKIP_SGLANG_LOG_FORMATTING"):
```

Comment:
Two nits:

1. Would you ever or a user ever reasonably set `DYN_SKIP_SGLANG_LOG_FORMATTING=0`? If so, probably want something like:
```
if os.environ.get("DYN_SKIP_SGLANG_LOG_FORMATTING", "0") in ["yes", "true", "1"]:
```

2. Double negative can be a little confusing. Do you want to opt-in or opt-out? If opt-out, would it make sense to be something like `DYN_ENABLE_SGLANG_LOGGING=0` (default is true)? Just thinking an alternative to "if not skip" logic